framework/include/meshgenerators/MeshGenerator.h

```diff
@@ -437,6 +450,12 @@ class MeshGenerator : public MooseObject, public MeshMetaDataInterface
 
   /// A user-defined name to save the mesh
   const std::string & _save_with_name;
+
+  /// Whether or not this mesh generator is a subgenerator
```

Comment:
is this the name of the parent instead of the child?

Reply:
nope, commented below

Reply:
still not a boolean